docs/devices/4512737_4512738.md

```diff
@@ -23,8 +23,21 @@ pageClass: device-page
 
 
 <!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+### Exposing floor sensor in Home Assistant UI
```

Comment:
If I understand correctly, this does not expose a new sensor but updated the thermostat entity to use the `outdoor_temperature`, if yes, please explain that here.

Reply:
I would replace "Exposing" with Using. Correct, this is just populating climate entity with `outdoor_temperature` insted of `local_temperature` .